src/frontend/js/modules/search/header/search-header.component.spec.js

```diff
@@ -69,4 +71,21 @@ describe('The esnSearchHeader component', function() {
 
     expect($state.go).to.have.been.calledWith('search.main', { a: null, p: null, q: 'cow' }, { location: 'replace', reload: true });
   });
+
+  it('should show global search box if showGlobalSearch is true', function() {
+    scope = $rootScope.$new();
+    compileSearchHeaderDirective();
+
+    expect(element.find('.search-header').length).to.not.equal(0);
```

Comment:
Not equal 0 can be negative numbers :trollface: 

```suggestion
    expect(element.find('.search-header').length > 0).to.be.true;
```